File: stock-market-service/src/main/java/rs/edu/raf/dto/StockDailyDTO.java
package rs.edu.raf.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Map;

public record StockDailyDTO(
    @JsonProperty("Meta Data")
    Map<String, String> metaData,
    @JsonProperty("Time Series (Daily)")
    Map<String, StockHistoryInfoDTO> timeSeriesDaily
) {}

File: stock-market-service-kotlin/src/main/kotlin/org/example/stockmarketservicekotlin/dto/StockDailyDTO.kt
package org.example.stockmarketservicekotlin.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class StockDailyDTO(@JsonProperty("Meta Data")
                         val metaData:Map<String, String>,
                         @JsonProperty("Time Series (Daily)")
                         val timeSeriesDaily: Map<String, StockHistoryInfoDTO>)
